script.js

```diff
@@ -1,4 +1,489 @@
+  }
+  if (secondDraw.name == "ace" && gameMode == "p1") {
+    gameMode = "chooseAce";
+    p1StartHand = initialHand;
+    return `<b>You drew ${firstDraw.name} of ${firstDraw.suit} and ${secondDraw.name} of ${secondDraw.suit}<br><br>Choose 1 or 11 to represent your ACE.`;
+  }
+  if (gameMode == "p1") {
+    p1StartHand = initialHand;
+    gameMode = "p1HOS";
+    return `<b>You drew ${firstDraw.name} of ${firstDraw.suit} and ${secondDraw.name} of ${secondDraw.suit}<br><br>Your starting hand is ${p1StartHand}!`;
+  }
+  if (gameMode == "pc") {
+    gameMode = "gameboard";
+    while (pciHand <= 17) {
+      pciHand = Number(String(pciHand)) + additionalDraw.rank;
+    }
+    return `<b>PC's final hand is ${pciHand}!`;
+  }
+};
+
+var aceOnHand = function (input) {
+  if (firstDraw.name == "ace") {
+    if (input == 1) {
+    }
+    if (input == 11) {
+      p1StartHand = Number(String(p1StartHand)) + 10;
+    }
+  }
+  if (secondDraw.name == "ace") {
+    if (input == 1) {
+    }
+    if (input == 11) {
+      p1StartHand = Number(String(p1StartHand)) + 10;
+    }
+  }
+  gameMode = "aceHOS";
+  return `<b>You have chose ${input} for your ACE.<br><br>Your starting hand is ${p1StartHand}`;
+};
+
+var hitOrStand = function (input) {
+  additionalDraw = shuffle();
+  if (gameMode == "p1HOS") {
+    if (input == "hit") {
+      p1StartHand = Number(String(p1StartHand)) + additionalDraw.rank;
+    }
+    if (input == "stand") {
+      gameMode = "pc";
+      return `<b>${playerName}'s final hand is ${p1StartHand}.<br><br>Computer's turn.`;
+    }
+  }
+  if (gameMode == "aceHOS") {
+    if (input == "hit") {
+      p1StartHand = Number(String(p1StartHand)) + additionalDraw.rank;
+    }
+    if (input == "stand") {
+      gameMode = "pc";
+      return `<b>${playerName}'s final hand is ${p1StartHand}.<br><br>Computer's turn`;
+    }
+  }
+  return `<b>You draw an additional ${additionalDraw.name} of ${additionalDraw.suit}.<br>Your current hand is ${p1StartHand}.<br><br>Please type stand if you've burst or if you wish to finalize your hand!<br>Please type hit again to draw somemore additional cards`;
+};
+
+var board = function () {
+  if (gameMode == "gameboard") {
+    /// both draw ace and player wins
+    if (firstDraw.name == "ace" && secondDraw.name == "ace" && pciHand < 21) {
+      p1winCounter = p1winCounter + 1;
+      playingChips = playingChips + Number(String(playerBets));
+      return `<b>${playerName}'s final hand = ${blackjack}<br>Computer's final hand = ${pciHand}.<br><br>Since ${playerName}'s hand is bigger than computer's.<br>Therefore ${playerName} wins!✌️<br>Earlier you bet ${playerBets} chips<br>Therefore you won ${playerBets} chips<br><br>${playerName} currently has ${playingChips} chips<br><br>${playerName} has won ${p1winCounter} round<br>Computer has won ${pcWinCounter} round<br>Draw for ${drawCounter} round<br><br>Click the submit button to play ♠ again!`;
```

Comment:
is there a condition that double aces would cause the player to win?
it seems like we are granting the win to the player without comparing with what the computer has

Reply:
based on my playing experience double ace = blackjack , i think there're diff kind of rules